app/components/primer/beta/auto_complete.rb

```diff
@@ -5,13 +5,16 @@ module Beta
     # Use `AutoComplete` to provide a user with a list of selectable suggestions that appear when they type into the
     # input field. This list is populated by server search results.
     # @accessibility
-    #   Always set an accessible label to help the user interact with the component.
+    #   * Always set an accessible label to help the user interact with the component.
     #
-    #   * Set the `label` slot to render a visible label. Alternatively, associate an existing visible text element
+    #     * Set the `label` slot to render a visible label. Alternatively, associate an existing visible text element
     #   as a label by setting `aria-labelledby`.
-    #   * If you must use a non-visible label, set `:"aria-label"` on `AutoComplete` and Primer
+    #     * If you must use a non-visible label, set `:"aria-label"` on `AutoComplete` and Primer
     #   will apply it to the correct elements. However, please note that a visible label should almost
     #   always be used unless there is compelling reason not to. A placeholder is not a label.
+    #
+    #   * Provide a `div` element with an `id` of `<list_id>-feedback` and class of `sr-only` (`list_id` should match `list_id` passed in to the initial component) to provide feedback to screen reader users about the available autoselect options.
```

Comment:
Can we render this entirely for the user instead? Thinking: we have an ID for the listbox, so let's render the `div` for them.

Reply:
Oooh I love that idea! Is that something we would do as part of a different Issue since this one was purely for updating docs? Perhaps it's not too complicated and can just be done in this file 🤔